x-pack/plugins/search_notebooks/common/constants.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { Notebook } from './types';
+
+export const INTRODUCTION_NOTEBOOK: Notebook = {
```

Comment:
@serenachou @leemthompo Could use some extra 👀  on the copy here.

Reply:
> we'll also want to do a pass on all the descriptions in [notebook_catalog](https://github.com/elastic/kibana/blob/057a72a636d4ada5d1c78959cc2043c90625866f/x-pack/plugins/search_notebooks/server/lib/notebook_catalog.ts) some time too

I can create an issue to work on that copy